app/src/main/kotlin/com/camerasecuritysystem/client/SettingsActivity.kt

```diff
@@ -46,30 +115,43 @@ class SettingsActivity : AppCompatActivity(),
         return false
     }
 
-    override fun applyTexts(cameraID: String?, port: String?, ipAddress: String?, password: String?) {
+    override fun applyTexts(
```

Comment:
This doesnt change anything and will conflict...

Reply:
Removed the change